packages/core/contracts/external/polygon/lib/ExitPayloadReader.sol

```diff
@@ -0,0 +1,160 @@
+pragma solidity ^0.8.0;
+
```

Comment:
I'm assuming all contract changes are pure copy-paste? Or did you make modifications?

Reply:
Pure copy-paste, check the change log